apps/meteor/server/services/omnichannel/queue.ts

```diff
@@ -123,26 +126,74 @@ export class OmnichannelQueue implements IOmnichannelQueue {
 		void (routingSupportsAutoAssign ? this.start() : this.stop());
 	}
 
+	private async reconciliation(reason: 'closed' | 'taken' | 'missing', { roomId, inquiryId }: { roomId: string; inquiryId: string }) {
+		switch (reason) {
+			case 'closed': {
```

Comment:
Also, I think we can use the same `case` for `closed` and `missing`, they are almost identical, just the log message is slightly different. WDYT?

Reply:
Message is different. They call the same funciton but this way we know what actually happened with a room by seeing the logs.